### ISSUE TITLE
wayland-wall: EOL?

### ISSUE THREAD

**Issue by mathstuf** [sardemff7/eventd#54]
The `nd-wayland` plugin requires `wayland-wall`. The official `wayland-wall/wayland-wall` repository seems to be gone and only forks are left (none of which have official source releases). Are there potential replacements to use instead?

**Comment by sardemff7** [comment]
Yeah, Wayland Wall wasn’t getting any traction and lost against wlroots’ layer-shell, and I didn’t have enough energy to keep working on Wayland Wall just for myself.

I was originally planning on re-using the code with a libweston plugin shipped with eventd and removing `nd` backends by using Weston as a compatibility layer, but I never got around to doing that. (And it’s a somewhat silly idea, honestly, but fun to imagine.)

Were you using that backend or is that just curiosity?

**Comment by mathstuf** [comment]
> Were you using that backend or is that just curiosity?

I was just back in my `eventd` packaging and wanted to check up on the status of the `nd-wayland` plugin that's behind a flag.
